src/index.html

```diff
@@ -2,11 +2,322 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width,
+      initial-scale=1.0"
+    >
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="catalog-page">
+    <header class="header catalog-page__header">
+      <a class="header__logo" href="#Home">
+        <img
+          class="header__image"
+          src="./images/logo.png"
+          alt="Moyo logo."
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="is-active nav__link" href="#apple">
+              APPLE
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#samsung">SAMSUNG</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#telephone">СМАРТФОНЫ И ТЕЛЕФОНЫ</a>
+          </li>
+          <li class="nav__item" data-qa="nav-hover">
+            <a class="nav__link" href="computers">
+              НОУТБУКИ И КОМПЬЮТЕРЫ
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#gadgets">ГАДЖЕТЫ</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#tablets">ПЛАНШЕТЫ</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#foto">ФОТО</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#video">ВИДЕО</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <div class="products">
+        <div
+          class="products__product-card products__product-card--p--is-first-in-row">
```

Comment:
```suggestion
          class="products__product-card">
```
Bad practice to use such classes